### ISSUE TITLE
[Chips]  MDCChipView fails accessibility tests

### ISSUE THREAD

**Issue by material-mirror** [material-components/material-components-ios#6414]
This was filed as an internal issue. If you are a Googler, please visit [b/123068070](http://b/123068070) for more details.

---
#### Internal data
- Associated internal bug: [b/123068070](http://b/123068070)

**Comment by material-mirror** [comment]
The internal issue [b/123068070](http://b/123068070) is now closed. This issue is being closed as a result.
